package-manager/README.md

```diff
@@ -0,0 +1,159 @@
+
+#### Legacy Environment Variables
+
+| Legacy Variable          | Preferred Equivalent    | Notes         |
+|--------------------------|-------------------------|---------------|
+| `RSPM_LICENSE`           | `PPM_LICENSE`           | Same behavior |
+| `RSPM_LICENSE_SERVER`    | `PPM_LICENSE_SERVER`    | Same behavior |
+| `RSPM_LICENSE_FILE_PATH` | `PPM_LICENSE_FILE_PATH` | Same behavior |
+
+**Note:** Legacy `RSPM_` variables are fully supported but are planned for deprecation after 2025. For more details and updates, see the [Posit Package Manager release notes](https://docs.posit.co/rspm/news/). For new deployments, always use the `PPM_` prefix to ensure forward compatibility.
+
+### Volume Mounts
+
+For persistent data, add these volume mounts to your `docker run` command:
+
+```bash
+-v /data/rstudio-pm:/var/lib/rstudio-pm \
+-v /data/rstudio-pm-config:/etc/rstudio-pm
+```
+
+| Mount Point           | Description               |
+|-----------------------|---------------------------|
+| `/var/lib/rstudio-pm` | Package data and database |
+| `/etc/rstudio-pm`     | Configuration files       |
+
+### Custom Configuration
+
+Mount a custom configuration file:
+
+```bash
+docker run -v /path/to/rstudio-pm.gcfg:/etc/rstudio-pm/rstudio-pm.gcfg ...
+```
+
+See the [configuration documentation](https://docs.posit.co/rspm/admin/appendix/configuration/) for available options.
+
+## Exposed Ports
+
+| Port | Description |
+|------|-------------|
+| 4242 | HTTP web interface and API |
+
+## User
+
+Runs as the `rstudio-pm` user (UID/GID 999).
+
+## Differences from rstudio/rstudio-package-manager
+
+This image differs from the legacy [`rstudio/rstudio-package-manager`](https://hub.docker.com/r/rstudio/rstudio-package-manager) image:
+
+| Aspect           | This Image                             | rstudio/rstudio-package-manager                                                  |
+|------------------|----------------------------------------|----------------------------------------------------------------------------------|
+| Registry         | `posit/package-manager`                | `rstudio/rstudio-package-manager`                                                |
+| License env vars | `PPM_` prefix                          | `RSPM_` prefix                                                                   |
+| Variants         | `std` (with R/Python), `min` (minimal) | Single variant; multiple tags for different R/Python versions (not simultaneous) |
+| Base OS options  | Ubuntu 22.04, Ubuntu 24.04             | Ubuntu 22.04                                                                     |
+
+## Caveats
+
+### Security
+
+These images should be reviewed before production use. Organizations with specific CVE or vulnerability requirements should rebuild these images to meet their security standards.
```

Comment:
We can mention the rebuild frequency here I think. Maybe throw in some more stuff about patch versions overwriting any previous version for builds since the previous patch version's tags would go dormant when that happens.